MEIOUandTaxes1/src/localisation/replace/national_ideas_l_english.yml

```diff
@@ -5718,22 +5718,22 @@
- serbian_ideas_2: "Srebro i Sasi" 
+ serbian_ideas_2: "Silver and Saxons" 
  serbian_ideas_2_desc: "When the Saxons fled the Tatars in Translyvania, they found new homes in our lands. With them, they brought their experience of working the great silver mines of Hungary, founding and expanding many such mines in our own realm. Later rulers would see the value in this, granting them various privileges and inviting more. With this great source of revenue, we now possess the ability to do much good for our realm, and it is our duty to spend it wisely. New towns will be founded, armies will be raised, and our power ensured, now and forever. \nMine Output: §G+10%§! \nAmenities Infrastructure Cost: §G-10%§!"
- serbian_ideas_3: "Mac i medene reci" 
+ serbian_ideas_3: "Sword and words of honey" 
  serbian_ideas_3_desc: "With the many threats surrounding our realm, our rulers had to be flexible, to take opportunities as they came, and adeptly justify any breaches in etiquette. As time passed, this became the norm, something expected of all men of great power. To act with an unsheathed blade and words of honey, to stab a man and convince him it was not only justified, but fair."
- serbian_ideas_4: "Zakonopravilo i Dušanov zakonik"
+ serbian_ideas_4: "Dusan's Codex"
  serbian_ideas_4_desc: "With Dusan's Codex, built upon the great Saint Sava's own work, our empire at last has laws that truly meet its needs. The rights of our subjects, from the lowest peasant to the highest lord, have been clarified and separated, finally ensuring that the independent communities of the highlands submit to a ruler's will. Despite this, the worst excesses of our nobility have been curbed, and our subjects know it was through our kindness.\nNobility Power Modifier: §Y+10%§!\nTopography Impact on CE: §G-20%§!"
- serbian_ideas_5: "Zapad istoku i istok zapadu"
+ serbian_ideas_5: "West to east and east to west"
  serbian_ideas_5_desc: "Through our lands flow trade from both east and west, down rivers, through hills and valleys. Ragusans, Greeks, Bulgarians, Serbs, Latins, Albanians, and Vlachs, all gather, seeking a share of this lucrative trade, and we welcome them all.\nMerchants from Trade Centers: §G+5%§!"
- serbian_ideas_6: "Zemlja manastira"
+ serbian_ideas_6: "Land of monasteries"
  serbian_ideas_6_desc: "In our lands, it is the monasteries who hold the reins of religious power, and it is expected that any good ruler will patronize them such that they may guide the souls of our people without financial worry. With these funds, it is common for priests to produce works of scholarship the match of any in the Christian world, and many spread this knowledge to the devoted.\nClergy Power Modifier: §Y+10%§!\nKnowledge Output: §G+5%§!"
- serbian_ideas_7: "Alemanska i Vlaška Konjica"
+ serbian_ideas_7: "Alemannic-Vlach cavalry"
```

Comment:
The idea names here should be capitalized, since they're titles -- e.g. "Sword and Words of Honey".

Reply:
Fixed by 55b357ef1657c5e4a519f0d15db868573aac8bb6